velox/exec/SortBuffer.cpp

```diff
@@ -158,11 +164,18 @@ RowVectorPtr SortBuffer::getOutput() {
   return output_;
 }
 
+void SortBuffer::abort() {
```

Comment:
I think OrderBy can just reset sort buffer on abort. We don't need abort implementation here. Also note we shall have test coverage for each newly added API.